ast/policy.go

```diff
@@ -149,33 +149,34 @@ type (
 
 	// Comment contains the raw text from the comment in the definition.
 	Comment struct {
-		Text     []byte
-		Location *Location
+		Text     []byte    `json:"text"`
+		Location *Location `json:"location,omitempty"`
```

Comment:
While this is called out in the PR desc, I wanted to highlight it here too. This changes the JSON field names for comments to be consistent with out other ast nodes. What do people think about this?

I have done this since it seems to make sense that the comments are the same format as everything else, especially if the data is going to be read be a machine. However, we are breaking the API of the comments output here.

I think that we could leave it such that OPA parse remained the same with some more MarshalJSON options, but it's trickier... let me know what you think.

Reply:
Ok, if we're keen to keep the fields the same, and always include location in comment data from a parse, then I think the best thing to do here is to just drop the struct tags completely. I can create an issue for this.

Reply:
Perhaps this wasn't what I was thinking of. What annoyed me in the past is the capitalization of some attribute in the comments, while any other attribute in the AST is lowercased. Seems like you've done more here though. There's probably not that many people depending on AST serialization, but who knows 😬

Reply:
Yeah, perhaps best to leave it as is for now. It'd be nice to fix it one day, but perhaps best to leave it for now.